### DISCUSSION TITLE
Skinning TerriaMap

### DISCUSSION THREAD

**Discussion by hardreddata** [TerriaJS/terriajs#6540] [Q&A]
Hi Team,

Still running version 7 here. Every so often I like to check in an see how things are progressing.

In older instances of version 8 I could do this in my `global.scss` from the examples.

```
    .tjs-standard-user-interface__sidePanel {
        background: white;
    }
```

In the latest version I arrived at the below by inspecting the elements in the web browser.
```
    .SidePanelContainer-sc-133gx1j-0.hBqzAA {
        background: white;
    }
```
Am I doing it right?

The recent ability to style user uploaded geojson might get us over the line to upgrade.

Side note - earlier build issues on windows were solved with `yarn install` in place of `npm install`. I should have read the documentation.

Many thanks.

**Comment by zoran995** [comment]
Styling shouldn't be done that way as `SidePanelContainer-sc-133gx1j-0.hBqzAA` is generated dynamically during build time and your changes probably won't be working even between the minor versions of the codebase.
Depending on the use case if you can't go with a theme configuration, what can be done here is to extend the codebase to provide id selectors for components or add custom classes for components, that can be used for style override. I haven't used a theme config in terriajs so can't help with it.
at least those are the first ideas that come to my mind
